diff -up WiringPi-master/wiringPi/Makefile.1~ WiringPi-master/wiringPi/Makefile
--- WiringPi-master/wiringPi/Makefile.1~	2021-10-19 00:50:55.547738122 +0200
+++ WiringPi-master/wiringPi/Makefile	2021-10-19 00:52:00.785925683 +0200
@@ -75,7 +75,7 @@ static:
 
 $(DYNAMIC):	$(OBJ)
 	$Q echo "[Link (Dynamic)]"
-	$Q $(CC) -shared -Wl,-soname,libwiringPi.so$(WIRINGPI_SONAME_SUFFIX) -o libwiringPi.so.$(VERSION) $(OBJ) $(LIBS)
+	$Q $(CC) -shared -Wl,-soname,libwiringPi.so$(WIRINGPI_SONAME_SUFFIX) -o libwiringPi.so.$(VERSION) $(LDFLAGS) $(OBJ) $(LIBS)
 
 .c.o:
 	$Q echo [Compile] $<
